### PR TITLE
Update preview selector to use new styles

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Update preview selector to use new styles

[stage-2]

## Motivation and Context
Consistent behaviour and improved user experience

## How Has This Been Tested?
Tested changes locally.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc this is the initial PR for this. I just want to get it out there for you to try it out and let me know if you have any feedback on the functionality or styling. Will work on unit tests and a releasable version next. Thanks.

File: web/partials/launcher/apps-home.html [hunk 1]
```diff
@@ -10,32 +10,58 @@
...
+              tooltip-digest-on-resize
+            >
+            	<div class="row-entry">
+                <span class="text-muted mr-auto" ng-if="!selectedSchedule">Create or select a schedule</span>
+                <span class="u_text-ellipsis mr-auto" ng-if="selectedSchedule">{{selectedSchedule.name}}</span>
+              	<span class="text-muted pl-2"><i class="fa fa-sort"></i></span>
+              </div>
+            </button>
+          </ng-show>
+          <button id="preview-selector-dummy" class="form-control btn-select btn-schedule-selector flex-row" ng-show="tooltipKey">
```
Comment (ezequielc) [inline web/partials/launcher/apps-home.html:57]: @alex-deaconu I am not following why we need the hide/show based on `tooltipKey` and the dummy element here. Could you clarify? Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu Code looks good, just one comment inline.
Also, re style, the Selector popup is very tall for me. Not sure if you already finalized styling, but I would suggest setting it to the same height as the other Schedules popup. Thanks!

File: web/partials/launcher/apps-home.html (hunk 1 above)
Comment (alex-deaconu) [reply web/partials/launcher/apps-home.html:57]: @ezequielc I'm wondering the same, removed in a subsequent PR. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc good catch, I will look into the height issue. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

**Pull request merged**